lib/widgets/chatbot/core/chat_window.dart

```diff
@@ -282,11 +316,13 @@ class _InteractiveChatWindow extends State<InteractiveChatWindow> {
         Provider.of<ChatProvider>(context, listen: false).getLastMessage();
     userMessage['convoID'] = convoID;
 
-    String textJSON = jsonEncode(<String, String>{'text': "${userMessage['text']}"});
+    String textJSON =
+        jsonEncode(<String, String>{'text': "${userMessage['text']}"});
 
     channel.sink.add(textJSON);
 
-    if (Provider.of<ConfigProvider>(context, listen: false).getMode() != Mode.dev) {
+    if (Provider.of<ConfigProvider>(context, listen: false).getMode() !=
+        Mode.dev) {
       FirebaseDbService.addMessageData(userMessage);
     } else {
       print(userMessage);
```

Comment:
This needs to be changed to address the new way we are modeling conversations. Remember that we are not combining messages anymore, unless it is to send them to the chatbot.

However, this requires remodeling how conversations are managed in `ChatProvider`, and in consequence, all across the app. I realized that is kind of a massive and difficult task that requires good knowledge of the app, so I will write and push changes that remodels how `ChatProvider` works as well as how the rest of the app interacts with `ChatProvider`.